index.js

```diff
@@ -247,6 +247,11 @@ const process = function(event, callback, teamConfig) {
           const value2 = { team_slug: teamSlug, annotation_id: resp.annotation.id, mode: 'bot' };
           const message = { text: t('project_set') + ': ' + projectUrl, response_type: 'in_channel', token: ACCESS_TOKEN, channel: event.channel };
 
+          // Store SmoochUserSlackChannelUrl
+          setSmoochUserSlackChannelUrl(event, { teamId: teamConfig.teamId, id: resp.annotation.dbid }, config.checkApi.apiKey, callback, function(resp) {
+            console.log('Added smooch user slack channel url to smooch user annotation');
```

Comment:
Please add some information to the log that we can identify the object... for example, the annotation id

Reply:
Done